Fixes and PEP-8 improvements

Nanush7 opened the pull request:
- Minor fixes.
 - Minor PEP-8 improvements.

Nanush7 committed:
Fixes and PEP-8 improvements

dieg0moraes commented on logic/client.py:
```diff
@@ -3,7 +3,7 @@
 import asyncio
 from sys import platform
-from timeit import default_timer
+# from timeit import default_timer
```
Este comentario borrarlo nomás

dieg0moraes commented on logic/client.py:
```diff
@@ -13,10 +13,10 @@
 # Solution to issue #5
 if platform == "darwin":
```
Esto lo podemos chequear en el main? Y que pase por parámetro?

Nanush7 replied:
Podría ser, pero cuando se use por otro lado no funcionaría. Capaz podría ir a la clase Fuzzer. Y que el usuario pueda forzar el ssl_check.

Nanush7 replied:
Cambiado en 8a65866

dieg0moraes commented on logic/fuzzer.py:
```diff
@@ -142,16 +138,22 @@ def run(self, interval_count: int = 0, no_stop: bool = False):
+            self.logger.lerr("I've been interrupted :(")
+        finally:
+            # Always wipe tasks and urls.
+            future.cancel()
```
Nice

dieg0moraes reviewed (commented):
Cheto, borra ese comment nomás y ta

Nanush7 committed:
Asyncio loop creation fix in progress

Nanush7 committed:
Ready.
Changes:
 - Added checkSSL argument.
 - Updated README.
 - Statistics to dictionary.
 - Added timeout to statistics.

Nanush7 commented:
@dieg0moraes
Timeout ya no pasa como excepción, y lo agruegué a las stats. Después tengo que hacer lo de incrementar el tiempo.
Pasé las stats a un diccionario, así se puede iterar por el diccionario. Por ejemplo en fuzzer.print_stats() o stats.reset_stats().
Agregué el argumento --checkSSL a main.py para que el usuario pueda elegir si quiere checkear o no. Por defecto se determina según el OS, como hasta ahora.

Nanush7 committed:
Fixed typos
